0.preprocess-sites/site_processing_config.yaml

```diff
@@ -32,7 +32,7 @@ core:
 ---
 prefilter:
   perform: true
-  example_site: CP074A_A1-Site_1
+  example_site: CP123_A1_1
```

Comment:
How about PLATE01_A1_1 instead?
(See reasoning above)
(Well A1 Site 1 would be the first well and site of any hypothetical plate so that doesn't reveal anything)